### PR TITLE
Add Vehicle Attributes training code

### PR DESCRIPTION
**Pull request by elenash**
**Before submitting a pull request**, please make sure the following is done:

1. You have merged in the latest changes from `develop`.
2. Resolved any conflicts that may have happened from that merge.
3. Ensure that you have run all tests and linting locally.

### CODE REVIEW

**Comment by elenash** [comment]
@alexey-trushkov @snosov1 Please review

File: training_toolbox/vehicle_attributes/infer.py [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+        font = cv2.FONT_HERSHEY_PLAIN
+        cv2.putText(img, vtype, (35, 15 + 60 * it), font, 1, (0, 0, 0), 2, cv2.LINE_AA)
+        cv2.putText(img, gttype, (35, 35 + 60 * it), font, 1, (0, 0, 0), 2, cv2.LINE_AA)
+        it += 1
+        summ_temp += l2diss
+      summ_temp /= it
+      summ += summ_temp
+      pic += 1
+      cv2.imshow("example", img)
+      cv2.waitKey(0)
```
Comment (alexey-trushkov) [inline training_toolbox/vehicle_attributes/infer.py]: Please add event handler for pressing `Esc` key for correct exit.

File: training_toolbox/vehicle_attributes/eval.py [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+    if latest_checkpoint != new_checkpoint:
+      latest_checkpoint = new_checkpoint
+
+      print('\nEvaluating {0}'.format(latest_checkpoint))
+      print('=============================================================')
+
+      start = time.time()
+
+      eval_results = estimator.evaluate(input_fn=eval_data.input_fn, name='val')
+      print(eval_results)
```
Comment (alexey-trushkov) [inline training_toolbox/vehicle_attributes/eval.py]: Please add printing evaluation metrics in formatted manner.

**Comment by elenash** [comment]
@alexey-trushkov fixed comments

File: training_toolbox/vehicle_attributes/README.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+
+We provide predefined configuration for: 
+* Vehicle type and color recognition.
+  - Configuration file: [training_toolbox/vehicle_attributes/boxcars116k/config.py](boxcars116k/config.py).
+  - Trained model: [Vehicle Attributes](https://download.01.org/openvinotoolkit/training_toolbox_tensorflow/models/vehicle_attributes/type_color/vehicle-attributes-barrier-0103.tar.gz).
+    A model has two output nodes:
+    . type is 4 components: one for each of 4 types - car, van, truck, bus. The resulting type is argmax of these values.
+    . color is 3 components: LAB format normalized from 0 to 1
+
+For demonstration purpose only a small dataset is uploaded [Boxcars-116k](https://download.01.org/openvinotoolkit/training_toolbox_tensorflow/datasets/boxcars116k.tar.gz).
```
Comment (snosov1) [inline training_toolbox/vehicle_attributes/README.md]: The name of the dataset is misleading. Let's change it.

Comment (elenash) [reply training_toolbox/vehicle_attributes/README.md]: @snosov1 changed to cars_100

Comment (snosov1) [reply training_toolbox/vehicle_attributes/README.md]: Lena, the new link doesn't work. Is this intended?

**Comment by AlexanderDokuchaev** [comment]
https://github.com/opencv/openvino_training_extensions/pull/52

**Pull request closed without merging**